applications/dashboard/views/entry/connect.php

```diff
@@ -117,24 +117,31 @@
                         wrap(sprintf(t('ConnectRegisteredName', 'Your registered username: <strong>%s</strong>'), htmlspecialchars($Row['Name'])), 'div', ['class' => 'ExistingUsername']);
                         $this->addDefinition('NoConnectName', true);
                         echo $this->Form->hidden('UserSelect', ['Value' => $Row['UserID']]);
-                    } else {
-                        echo $this->Form->label('Username', 'ConnectName');
-                        echo '<div class="FinePrint">', t('ConnectChooseName', 'Choose a name to identify yourself on the site.'), '</div>';
+                    }
 
-                        if (count($ExistingUsers) > 0) {
-                            foreach ($ExistingUsers as $Row) {
-                                echo wrap($this->Form->radio('UserSelect', $Row['Name'], ['value' => $Row['UserID']]), 'div');
-                            }
-                            echo wrap($this->Form->radio('UserSelect', t('Other'), ['value' => 'other']), 'div');
+                    // Found User's Name in GDN_User.
+                    if (count($ExistingUsers) >= 1 && !$NoConnectName){
+                        echo $this->Form->label('Username', 'ConnectName');
+                        $connectInstructions = \Gdn::translate('One or more users with your name already exist, would you like to connect as them?');
+                        echo $connectInstructions;
+                        $connectNameMessage = (!$allowConnect) ? '' : \Gdn::translate(' <span class="FinePrint">(Requires a password)</span>');
```

Comment:
Jut make the translation string "(Requires a password)" and wrap it in the HTML.